docs/dmarc-builder.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: default
+title: CAA Builder
```

Comment:
Shouldn't this be DMARC builder?

Reply:
Yes, you are right. I thought I had corrected it, but apparently, I did not.

Reply:
It happens.

Does this mean that after it is fixed the doc should be live on the site?

Reply:
Thanks for pointing this out!

The docs are based on master.  I fixed this in master 1 minute ago, so the docs should update soon.

For future references, if you search https://stackexchange.github.io/dnscontrol/js for the word `dmarc_` you'll have found the docs, just in the wrong place.